### PR TITLE
Disallows `bookend-config-src` attr in `amp-story` tag

### LINKED ISSUE #15796: Remove bookend-config-src from validator for amp-story 1.0
We should disallow the `bookend-config-src` attribute on `<amp-story>` in amp-story 1.0 (in favor of the `<amp-story-bookend>` tag).

### PR DESCRIPTION
**Pull request by Enriqe**
Closes #15796

With the introduction of the bookend config API, we want to deprecate and eventually remove the ability to add a `bookend-config-src` attribute in the `amp-story` tag.

### CODE REVIEW

**Comment by newmuis** [comment]
I don't think I get how this does what it's saying.  Do we need to somewhere specify in the protoascii that `bookend-config-src` is deprecated?

Adding @Gregable because I clearly don't know what's going on here 😝

File: extensions/amp-story/validator-amp-story.protoascii [hunk 1]
```diff
@@ -48,6 +48,8 @@ tags: {  # <amp-story>
       allowed_protocol: "http"
       allowed_protocol: "https"
     }
+    deprecation: "AMP-STORY-BOOKEND"
+    deprecation_url: "https://www.ampproject.org/docs/reference/components/amp-story#announcements"
```
Comment (newmuis) [inline extensions/amp-story/validator-amp-story.protoascii]: ⚠️ Heads up: Your #15465 changes the name of this section, which will change the URL fragment as well.

Comment (Enriqe) [reply extensions/amp-story/validator-amp-story.protoascii]: Done.

**Comment by Enriqe** [comment]
@Gregable friendly ping

**Comment by Enriqe** [comment]
@newmuis read to merge

**Comment by newmuis** [comment]
Actually just to double-check:

This still allows `bookend-config-src` to be valid, but throws a deprecation warning, is that correct @Enriqe @Gregable?

**Comment by Enriqe** [comment]
Yes it will still be valid but throw a warning like so:

image

**Pull request merged**